clj-refactor.el

```diff
@@ -221,6 +221,15 @@ won't run if there is a broken namespace in the project."
   :group 'cljr
   :type 'boolean)
 
+(defcustom cljr-ignore-libspecs
```

Comment:
an ns is something else, a libspec may or may not contain a reference to a namespace.

The problem I'm having with the current name ist hat we're not really ignoring them we're just not pruning the occurrences :/